crypto/keys/src/x25519.rs

```diff
@@ -377,7 +377,7 @@ impl KexEphemeralPrivate for X25519EphemeralPrivate {
 
 impl FromRandom for X25519EphemeralPrivate {
     fn from_random<R: CryptoRng + RngCore>(csprng: &mut R) -> X25519EphemeralPrivate {
-        X25519EphemeralPrivate(EphemeralSecret::new(csprng))
+        X25519EphemeralPrivate(EphemeralSecret::random_from_rng(csprng))
```

Comment:
Can you comment a bit more on this change?

Reply:
They just renamed the function